src/test/func/malloc/malloc.cc

```diff
@@ -21,7 +21,7 @@ void check_result(size_t size, size_t align, void* p, int err, bool null)
     if (our_malloc_usable_size(p) < size)
       abort();
 
-    if (((uintptr_t)p % align) != 0)
+    if ((size_t)((uintptr_t)p % align) != 0)
```

Comment:
Please don't add C-style casts (and, if you're modifying code that contains them, please fix them).